src/openforms/js/components/admin/form_design/utils.js

```diff
@@ -122,6 +122,39 @@ const parseValidationErrors = (errors, prefix) => {
   return parsedErrors;
 };
 
+// FormioUtils.eachComponent doesn't give us any useful information, so let's implement
+// it similar to the backend.
+function* _componentWithPathGenerator(components, prefix = 'components') {
+  for (let index = 0; index < components.length; index++) {
+    const component = components[index];
+    const fullPath = `${prefix}.${index}`;
+    yield [fullPath, component];
+
+    // taken from FormioUtils.eachComponent
+    const hasColumns = component.columns && Array.isArray(component.columns);
+    // const hasRows = component.rows && Array.isArray(component.rows);
```

Comment:
This is commented out instead of removed in case we at some point enable the table component?

Reply:
correct!